apps/publish/enqueue/enqueue_service.py

```diff
@@ -466,9 +473,19 @@ def queue_transmission(self, doc, subscribers, subscriber_codes=None, associatio
             subscriber_codes = {}
 
         try:
+            if config.PUBLISH_ASSOCIATIONS_RESEND and not publish_first_version:
+                if config.PUBLISH_ASSOCIATIONS_RESEND == "corrections":
+                    self.resend_association_items(doc)
+                elif config.PUBLISH_ASSOCIATIONS_RESEND == "updates" and doc.get("state") not in [
+                    "corrected",
+                    "being_corrected",
+                ]:
+                    self.resend_association_items(doc)
```

Comment:
would it actually resend the associations if it's completely new item with already published associations?